README.md

```diff
@@ -2,143 +2,171 @@
 
-- Additions that strictly improve objective, numerical quality criteria (speedup, better browser support)
+- Additions that strictly improve objective, numerical quality criteria
+  (speedup, better browser support)
 - Fixing objectively incorrect behavior
 - Rephrasing, reorganizing or refactoring
 - Addition or removal of warnings
-- Additions only likely to be _noticed by_ other implementors-of-Astro, invisible to users-of-Astro.
+- Additions only likely to be _noticed by_ other implementors-of-Astro,
+  invisible to users-of-Astro.
 
-If you submit a pull request to implement a new feature without going
-through the RFC process, it may be closed with a polite request to
-submit an RFC first.
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
 
 ## Why do you need to do this
 
-It is great that you are considering suggesting new features or changes to Astro - we appreciate your willingness to contribute! However, as Astro becomes more widely used, we need to take stability more seriously, and thus have to carefully consider the impact of every change we make that may affect end users. On the other hand, we also feel that Astro has reached a stage where we want to start consciously preventing further complexity from new API surfaces.
+It is great that you are considering suggesting new features or changes to
+Astro - we appreciate your willingness to contribute! However, as Astro becomes
+more widely used, we need to take stability more seriously, and thus have to
+carefully consider the impact of every change we make that may affect end users.
+On the other hand, we also feel that Astro has reached a stage where we want to
+start consciously preventing further complexity from new API surfaces.
 
-These constraints and tradeoffs may not be immediately obvious to users who are proposing a change just to solve a specific problem they just ran into. The RFC process serves as a way to guide you through our thought process when making changes to Astro, so that we can be on the same page when discussing why or why not these changes should be made.
+These constraints and tradeoffs may not be immediately obvious to users who are
+proposing a change just to solve a specific problem they just ran into. The RFC
+process serves as a way to guide you through our thought process when making
+changes to Astro, so that we can be on the same page when discussing why or why
+not these changes should be made.
 
 ## Gathering feedback before submitting
 
-It's often helpful to get feedback on your concept before diving into the
-level of API design detail required for an RFC. **You may open an
-issue on this repo to start a high-level discussion**, with the goal of
-eventually formulating an RFC pull request with the specific implementation
-design.
+It’s often helpful to get feedback on your concept before diving into the level
```

Comment:
Same here

Reply:
https://www.fileformat.info/info/unicode/char/2019/index.htm
https://dictionary.cambridge.org/us/grammar/british-grammar/possession-john-s-car-a-friend-of-mine